test-cases/scylla-operator/longevity-scylla-operator-3h-gke.yaml

```diff
@@ -9,5 +9,5 @@ n_monitor_nodes: 1
 nemesis_class_name: 'KubernetesScyllaOperatorMonkey'
 nemesis_interval: 5
 
-user_prefix: 'k8s-longevity-10gb-3h-gke'
+user_prefix: 'long-10gb-3h'
```

Comment:
mb, long-10gb-3h-gke ?

Reply:
Don't need because it'll render to something like `long-10gb-3h-jenkins-k8s-gke-545231d6`.

Reply:
good, could you add this information to the comment near the prefix

Reply:
done